gliderpy/fetchers.py

```diff
@@ -30,6 +30,18 @@
     "platform_deployment",
 ]
 
+ooi_vars = [
```

Comment:
These will become hard to maintain as list. Maybe let's move them to a dictionary where the key is the server and the values are the variables list?

Reply:
Also resolved n a667bc8